src/dash-table/components/tooltipHelper.ts

```diff
@@ -0,0 +1,39 @@
+export default (
+    el: HTMLElement | null,
+    parent: HTMLElement | null = null
+) => {
+    parent = parent || (() => {
+        parent = el;
+
+        while (parent && parent.nodeName.toLowerCase() !== 'td') {
+            parent = parent.parentElement;
+        }
+
+        return parent;
+    })();
+
+    if (!el || !parent) {
+        return {};
+    }
+
+    let positionalParent = el;
+    while (getComputedStyle(positionalParent).position !== 'relative' &&
+        getComputedStyle(positionalParent).position !== 'sticky') {
+        if (!positionalParent.parentElement) {
+            break;
+        }
+
+        positionalParent = positionalParent.parentElement;
+    }
+
+    let relativeParent = el;
+    while (getComputedStyle(relativeParent).position !== 'relative') {
+        if (!relativeParent.parentElement) {
+            break;
+        }
+
+        relativeParent = relativeParent.parentElement;
+    }
+
+    return { positionalParent, parent };
```

Comment:
Finds out who is the relative parent and the real parent of the tooltip -- necessary to position it absolutely correctly.